File: packages/csharp/src/Anip.Core/CostActual.cs
using System.Text.Json.Serialization;

namespace Anip.Core;

public class CostActual
{
    [JsonPropertyName("financial")]
    public FinancialCost? Financial { get; set; }

    [JsonPropertyName("variance_from_estimate")]
    public string? VarianceFromEstimate { get; set; }
}

File: packages/java/anip-core/src/main/java/dev/anip/core/CostActual.java
package dev.anip.core;

/**
 * The actual cost incurred by an invocation.
 */
public class CostActual {

    private final FinancialCost financial;
    private final String varianceFromEstimate;

    public CostActual(FinancialCost financial, String varianceFromEstimate) {
        this.financial = financial;
        this.varianceFromEstimate = varianceFromEstimate;
    }

    public FinancialCost getFinancial() {
        return financial;
    }

    public String getVarianceFromEstimate() {
        return varianceFromEstimate;
    }
}
